.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: objective-c
+osx_image: xcode8.3
+xcode_sdk: iphonesimulator10.3
+xcode_workspace: workspace.xcworkspace
+xcode_scheme: test
+
+script:
+  - xcodebuild -workspace "workspace.xcworkspace" -scheme "test" clean build CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
+  - xcodebuild test -workspace "workspace.xcworkspace" -scheme "test" -destination 'platform=iOS Simulator,name=iPhone 7,OS=10.3' CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO 
```

Comment:
That works for now. I saw that a few other people online were having the same problem, but didn't see any real resolutions. Maybe once we're all more familiar with Xcode/CI we'll find a way to add it back in.